### DISCUSSION TITLE
allow startWith to take observable as input

### DISCUSSION THREAD

**Discussion by michaelbe812** [ReactiveX/rxjs#7301] [Q&A] [answered]
HI,
 in a lot of cases it would be quite handy that `startWith` could take another observable as initial value instead of just a simple value.

Example
```
const startValue$ = of(anyThing)

const source$ = some$.pipe(
  startWith(startValue$)
)
```

Does this feature request makes sense for you? Looking forward to feedback

**Comment by voliva** [comment, marked as answer]
`const source$ = concat(startValue$, some$)`
